.docker/debian/lean/Dockerfile

```diff
@@ -26,7 +26,9 @@ RUN python3 -m pip install --user mathlibtools
 # now install `pyinstaller`, and run it on the installed copy of `leanproject`
 RUN python3 -m pip install --user pyinstaller
 WORKDIR /home/lean/.local/bin
-RUN pyinstaller --onefile --noconfirm leanproject
+# If mathlibtools added an explicit `import _cffi_backend` or `import cffi`
+# we wouldn't need the `--hidden-import` flag here.
```

Comment:
```suggestion
# If PyNaCl added an explicit `import _cffi_backend` or `import cffi`
# we wouldn't need the `--hidden-import` flag here; see
# https://github.com/PyGithub/PyGithub/issues/2035
```

Reply:
Or even https://github.com/pyinstaller/pyinstaller/issues/3525 might be a better issue to reference (I've linked them together so it doesn't matter much either way)